packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -154,6 +161,12 @@ export async function produceBlockBody<T extends BlockType>(
     proposerSlashings: proposerSlashings.length,
   });
 
+  if (ForkSeq[fork] >= ForkSeq.altair) {
```

Comment:
Do you want to use `isForkLightClient(fork)` so it's aligned with `isForkExecution(fork)` at L171?

Reply:
sure we can but if you look further down we use the `ForkSeq`
https://github.com/ChainSafe/lodestar/blob/99794d3b261bb50f9a6c7d9f6ca19291b3f30911/packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts#L211

but since those are close together can use `isForkLightClient` here, doesn't matter much but wanna avoid too much refactoring in this PR